src/roots.jl

```diff
@@ -30,6 +30,9 @@ end
 #         end
 #     end
 # else
+complexroots(cfs::Vector{T}) where T = 
+    sort(eigvals(companion_matrix(chop(cfs, 10*eps(T)))), lt = (x, y) -> real(x) < real(y) ? true : (real(x) > real(y) ? false : (imag(x) < imag(y) ? true : false)), rev=true)
```

Comment:
Is this lexicographic ordering? In that case perhaps we might want to set the `by` keyword argument instead of the `lt` one, as `by = x->(real(x), imag(x))`